### ISSUE TITLE
Post Splitter Not Working.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#633]
```
What's the problem?

Post splitter for Blogger is not working.



What browser are you using?


Firefox 13


What version of ScribeFire are you running?
4
```

-----
Original issue reported on code.google.com by `abhishek...@gmail.com` on 22 Jun 2012 at 6:32

**Comment by GoogleCodeExporter** [comment]
```
same here
```

-----
Original comment by `corgaud...@gmail.com` on 17 Jul 2012 at 6:38

**Comment by GoogleCodeExporter** [comment]
```
Fixed for blogger blogspot in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 6:56
* Changed state: **Fixed**
